lib/tests/fixtures/facts/external/posix/execution/error_message

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env sh
+echo error message! >&2
```

Comment:
Yeah, was just going to add a batch + another test for the PS resolver specifically.